syft/frameworks/torch/federated/dataloader.py

```diff
@@ -110,6 +110,50 @@ def stop(self):
         raise StopIteration
 
 
+class _DataLoaderOneWorkerIter(object):
+    r"""Iterates once over the worker's dataset, as specified by its sampler"""
+
+    def __init__(self, loader, worker_idx):
+        self.loader = loader
+        self.federated_dataset = loader.federated_dataset
+
+        # Assign the worker to invoke
+        self.worker = loader.workers[worker_idx]
+
+        # The function used to stack all samples together
+        self.collate_fn = loader.collate_fn
+
+        # Create a sample iterator for each worker
+        self.sample_iter = iter(loader.batch_samplers[self.worker])
+
+    # def __len__(self):
```

Comment:
why is this commented out

Reply:
That is a left-over of some tests. I will un-comment it.